src/elements/content-explorer/ContentExplorer.js

```diff
@@ -795,10 +783,11 @@ class ContentExplorer extends Component<Props, State> {
 
         const selectedItem: BoxItem = { ...item, selected: true };
 
-        const { newCollection } = this.updateCollection(currentCollection, selectedItem);
+        this.updateCollection(currentCollection, selectedItem);
+
         const focusedRow: number = items.findIndex((i: BoxItem) => i.id === item.id);
 
-        this.setState({ currentCollection: newCollection, focusedRow, selected: selectedItem }, () => {
+        this.setState({ focusedRow }, () => {
```

Comment:
It seems like this callback should actually be used on L786 instead since it depends on `selectedItem` being set to state